R/path.R

```diff
@@ -57,7 +57,11 @@ standardise_one_path <- function (path, check = TRUE) {
     gz = gzfile(path, ""),
     bz2 = bzfile(path, ""),
     xz = xzfile(path, ""),
-    zip = zipfile(path, ""),
+    zip = if(check){
+            zipfile(path, "")
+          } else {
+            stop("Writing zip files is not supported")
```

Comment:
I think it would be a little cleaner to do the check in the `if (check)` block above.  But I think the `check` argument name is confusing, and it would be more informative if it was called `write`. Would you mind making that change too?

Reply:
I think `check` should be called `read` instead of `write` no?

Reply:
You'd have to flip the value, but I think `write` would make it more clear to me

Reply:
You'll also need to update the block that use `check` above, and that's where I'd recommend you also perform this check.

Reply:
I did in the commit https://github.com/r-lib/vroom/pull/155/commits/16db47d51130a70345a9d0f0ec4ac54cb41e2b3c

Reply:
Oh sorry, somehow github didn't show that to me.